documentation/book/proc-renewing-ca-certs-manually.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='proc-renewing-ca-certs-manually-{context}']
+
+= Renewing CA certificates manually
+
+If automatic CA certificate generation is enabled (`generateCertificateAuthority=true`), the cluster CA and clients CA certificates will auto-renew at the start of their respective certificate renewal periods. You can manually renew one or both of these certificates before the certificate renewal period begins, if required for security reasons. A renewed certificate uses the same private key as the old certificate.
+
+.Prerequisites
+
+* The Cluster Operator is running.
+* A Kafka cluster in which CA certificates and private keys are installed.
+
+.Procedure
+
+. View a description of the certificate authority `Secret` that contains the CA certificate that you want to renew.
+
+.. To view the `Secret` for the cluster CA certificate: 
++ 
+[source,shell,subs="+quotes"]
+kubectl describe secret _<cluster-name>_-cluster-ca-cert
+
+.. To view the `Secret` for the clients CA certificate
++ 
+[source,shell,subs="+quotes"]
+kubectl describe secret _<cluster-name>_-clients-ca-cert
+
+. Annotate the `Secret` by applying the `strimzi.io/force-renew` annotation.
+
+.. To annotate the `Secret` for the cluster CA certificate:
++
+[source,shell,subs="+quotes"]
+kubectl annotate secret _<cluster-name>_-cluster-ca-cert strimzi.io/force-renew=true
+
+.. To annotate the `Secret` for the clients CA certificate:
++
+[source,shell,subs="+quotes"]
+kubectl annotate secret _<cluster-name>_-clients-ca-cert strimzi.io/force-renew=true
+
+The Cluster Operator generates a new CA certificate for the `Secret` that you annotated.
```

Comment:
I think we should add note about it being done only in next reconciliation and only during the maintenance window (if configured).